data/presets/barrier/rope.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icon": "temaki-rope_fence",
+    "fields": [
+        "access"
+    ],
+    "moreFields": [
+        "wheelchair"
```

Comment:
I wouldn't oppose this optional field, but interestingly, currently there are zero features mapped with `barrier=rope` + `wheelchair=*`. :shrug:

Reply:
I just added it as it's listed as a useful combination [on the wiki](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Drope). I wouldn't be opposed to removing it though.

Reply:
ok, then let's remove it for now